docs: clarify spl token account creation handling for exchange integrations (#23286)
solana-labs/solana: docs/src/integrations/exchange.md

identify the token mint and account owner (main wallet address) of the affected
account.

Note that if a receiving account is created during the transaction, it will have no

> `preTokenBalance` entry as there is no data to decode the state from.  In this

Reviewer: ```suggestion
`preTokenBalance` entry as there is no existing account state.  In this
```
Abstract away a little more of the implementation?

Author: whoops mergify beat me here.  Good idea though. I'll add it to a follow up
